src/gen/conditions.rs

```diff
@@ -199,6 +203,20 @@ pub fn parse_args(
             }
             Ok(Condition::CreateCoin(puzzle_hash, amount, a.null()))
         }
+        SOFTFORK => {
+            if (flags & NO_UNKNOWN_CONDS) != 0 {
+                // We don't know of any new softforked-in conditions, so they
+                // are all unknown
+                Err(ValidationErr(c, ErrorCode::InvalidConditionOpcode))
+            } else {
+                match sanitize_uint(a, first(a, c)?, 4, ErrorCode::InvalidSoftforkCost)? {
+                    // the first argument represents the cost of the condition.
+                    // We scale it by 10000 to make the argument be a bit smaller
+                    SanitizedUint::Ok(cost) => Ok(Condition::Softfork(cost * 10000)),
```

Comment:
Do we really want to bias via a power of 10 rather than a power of 2?

Who's in charge here, the people or the computers? :)

Reply:
well, all of our costs and cost limits are aligned with base 10 numbers.
I don't have a strong opinion on what this factor should be. I based this on the existing condition costs, which are `1800000` and `1200000`, so we might even get away with a precision of 100k.